assets/lokomotive-kubernetes/bootkube/resources/charts/kube-apiserver/templates/kube-apiserver-disruption.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: policy/v1beta1
```

Comment:
I think this commit (https://github.com/kinvolk/lokomotive/pull/32/commits/4d112c0eab1f51cacf5ae454d32df4df87702f0f) needs a commit message a lot bigger. We should explain what steps we're taking to make updates reliable, link to the upstream SO_REUSEPORT issue and possible limitations.

Reply:
Sure, done: https://github.com/kinvolk/lokomotive/pull/32/commits/d1de22ac8481e48873c8da568843498d4fc2973d.

Reply:
Can we mark this as resolved?

Reply:
I think so, as @iaguis approved the PR.